aws/rhel-8.5-devel-aarch64/main.tf

```diff
@@ -1,7 +1,7 @@
 module "aws" {
   source = "../_base"
 
-  name             = "rhel-8.5-aarch64"
+  name             = "rhel-8.5-devel-aarch64"
```

Comment:
Can you please rename `devel` to `nightly`? At least that's the term that we use for RHEL 9.

About `devel` vs `nightly` - I went originally with `devel` but then I realized that our CI uses `nightly` composes rather than `development` composes so I went with `nightly` to prevent confusion.